sys/i386/conf/MINIMAL

```diff
@@ -41,6 +41,7 @@ makeoptions	WITH_CTF=1		# Run ctfconvert(1) for DTrace support
 
 options 	SCHED_ULE		# ULE scheduler
 options 	PREEMPTION		# Enable kernel thread preemption
+options 	VIMAGE			# Subsystem virtualization, e.g. VNET
```

Comment:
again, unrelated change.

Reply:
do you want me to create a separate pull request for that?

Reply:
spun out into #962